src/test/java/seedu/address/testutil/TypicalAppObjects.java

```diff
@@ -64,13 +65,12 @@
     public static final String KEYWORD_MATCHING_MEIER = "Meier"; // A keyword that matches MEIER
 
     // Manually added - Doctor details
-    public static final String DRGRAY = "Dr. Gray";
-    public static final String DRWHO = "Dr. Who";
-    public static final String DRHOW = "Dr. How";
-    public static final String DRSTRANGE = "Dr. Strange";
-    public static final String DRWHICH = "Dr. Which";
-    public static final String DRWHY = "Dr. Why";
-    public static final String DRWHEN = "Dr. When";
+    public static final Doctor DR_GREY = new DoctorBuilder().withName("Dr Meredith Grey").build();
```

Comment:
an update to the `TypicalAppObject`'s objects will probably require an update to the corresponding json (`typicalDoctorRecords.json`, `typicalAppointmentSchedule.json`) as well.